website/frontend/templates/index.html

```diff
@@ -107,19 +107,19 @@ <h2>{{ _("Plugin Support") }}</h2>
       var platform = navigator.platform, downloadURL, iconClass, infoText;
 
       if (/^Win/.test(platform) || /Windows/.test(navigator.userAgent)) {
-        downloadURL = "http://ftp.musicbrainz.org/pub/musicbrainz/picard/picard-setup-1.4.2.exe";
+        downloadURL = "http://ftp.musicbrainz.org/pub/musicbrainz/picard/picard-setup-2.0.exe";
         iconClass = "fa-windows";
-        infoText = "v1.4.2 for Windows";
+        infoText = "v2.0 for Windows 64-bit";
```

Comment:
Will there be no 32 bit build anymore?

Reply:
Nope. Not officially. We do not have the appropriate resources to build it. Appveyor doesnt have the needed VMs.

Reply:
Oh, ok. Didn't we use Appveyor also for building 32 bit libdiscid?